source/fuzz/fuzzer_pass_replace_adds_subs_muls_with_carrying_extended.cpp

```diff
@@ -73,4 +76,4 @@ void FuzzerPassReplaceAddsSubsMulsWithCarryingExtended::Apply() {
   }
 }
 }  // namespace fuzz
-}  // namespace spvtools
+}  // namespace spvtools
```

Comment:
Make sure the file ends in a new line. There is a CLion option to add this on file save.